js/docx/smoke/api_rgb_color/to_json.js

```diff
@@ -0,0 +1,44 @@
+builder.CreateFile("docx");
+    let oDocument = Api.GetDocument();
+    let oParagraph = oDocument.GetElement(0);
+    let oRGBColor = Api.CreateRGBColor(255, 111, 61);
+    let jsonRGBColor = oRGBColor.ToJSON();
+        oParagraph.AddText(jsonRGBColor);
+GlobalVariable["JSON_RGBColor"] = jsonRGBColor;
+builder.CloseFile();
+
+
+/////////////////////
+
+
+builder.CreateFile("docx");
+    let oNewDocument = Api.GetDocument();
+    let jsonRGBColor = GlobalVariable["JSON_RGBColor"];
+    let oParagraphRGBColor = oNewDocument.GetElement(0);
+        oParagraphRGBColor.AddText(jsonRGBColor);
+
+    let oRGBColorFromJSON = Api.FromJSON(jsonRGBColor);
+    let oParagraphRGBColorObject = Api.CreateParagraph();
+        oParagraphRGBColorObject.AddDrawing(oRGBColorFromJSON);
```

Comment:
RGBColor is not a drawing! If you manage to add something to a paragraph using AddDrawing method passing RGBCorol, it is a bug. 

RGBColor is used as part of the fill

[Study the documentation for fill obj CreateSolidFill](https://api.onlyoffice.com/docbuilder/textdocumentapi/api/createsolidfill)